### DISCUSSION TITLE
trace view: support span status/kind/event

### DISCUSSION THREAD

**Discussion by hanjm** [grafana/clickhouse-datasource#335] [Ideas]
#329 has a awsome starting about support for trace, current it does not support 
1. [status](https://opentelemetry.io/docs/concepts/signals/traces/#span-status)
2. [spanKind[(https://opentelemetry.io/docs/concepts/signals/traces/#span-kind)
3. [span event](https://opentelemetry.io/docs/concepts/signals/traces/#span-events).

**Comment by SpencerTorres** [comment]
Any missing fields would need to be added to the visualization panel by Grafana here:
https://grafana.com/docs/grafana/latest/explore/trace-integration/#data-api

With `4.x` we have added all of the fields we can when visualizing a trace
